rclpy/rclpy/node.py

```diff
@@ -1676,6 +1676,7 @@ def create_timer(
         callback: Callable,
         callback_group: Optional[CallbackGroup] = None,
         clock: Optional[Clock] = None,
+        autostart: Optional[bool] = True,
```

Comment:
Maybe it is better just to do?
```suggestion
        autostart: bool = True,
```
As I get it from the description of the `Optional` [here](https://docs.python.org/3/library/typing.html#typing.Optional), it should represent a parameter with default `None`. I'd do it the same way in `timer.py`, in the `__init__` parameters section, [here](https://github.com/ros2/rclpy/pull/1138/files#diff-0860061fc25c955062f5fde6aeafebddccc61ebb3728746c781521106b0937e6R38).

Reply:
Yes, you are totally correct.  Fixed in 14192f30c7313543868b48b562367b32ff5f26c9